README.md

```diff
@@ -461,8 +464,7 @@ for_window [title="^Syncthing Tray( \(.*\))?$"] floating enable, border none, re
 The dark mode introduced in Windows 10 does not affect traditional desktop
 applications like Syncthing Tray. Unfortunately, Windows normal theming
 capabilities [have been
-removed](https://superuser.com/questions/949920/window-color-and-appearance-
-removed-in-win10). So far Qt does not offer much to improve the situation (see
+removed](https://superuser.com/questions/949920/window-color-and-appearance-removed-in-win10). So far Qt does not offer much to improve the situation (see
```

Comment:
It makes sense not to wrap the link but I'd still prefer to wrap the rest of the line.